core/src/main/java/org/infinispan/stream/impl/ClusterStreamManagerImpl.java

```diff
@@ -452,8 +452,8 @@ public void forgetOperation(Object id) {
       }
 
       @Override
-      public void subscribe(Subscriber<? super V> s, Consumer<? super Supplier<PrimitiveIterator.OfInt>> onLostSegments,
-            Consumer<? super Supplier<PrimitiveIterator.OfInt>> onSegmentsComplete) {
+      public void subscribe(Subscriber<? super V> s, Consumer<? super Supplier<PrimitiveIterator.OfInt>> onSegmentsComplete,
+            Consumer<? super Supplier<PrimitiveIterator.OfInt>> onLostSegments) {
```

Comment:
`RemoteIteratorPublisher` default subscribe method still uses the old parameter order (although neither supplier does anything).

Reply:
Yeah this wasn't a bug, just variables in wrong spots :) I can fix the other.